Sources/WalletConnectSign/NetworkInteractor/NetworkInteractor.swift

```diff
@@ -62,18 +59,6 @@ class NetworkInteractor: NetworkInteracting {
         setUpPublishers()
     }
 
-    func request(_ wcMethod: WCMethod, onTopic topic: String) async throws {
```

Comment:
the idea was replace `request` method with async `request`. But here you removed all async calls

Reply:
totally agree, do not get the purpose of doing so.